test/e2e/minishift.test.ts

```diff
@@ -9,12 +9,17 @@
  **********************************************************************/
 import { expect, test } from '@oclif/test'
 
+import { E2eHelper } from './util/e2e'
+
+const helper = new E2eHelper()
 jest.setTimeout(600000)
 
-describe('e2e test', () => {
-  describe('server:start without parameters', () => {
+const execution_platform = 'openshift'
```

Comment:
To me just `PLATFORM` looks better. But up to you.

Reply:
I like more PLATFORM!